src/components/CommentForm.jsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from "react";
+import { postComment } from "../utils";
+
+export function CommentForm({
+  newComment,
+  currentAuthor,
+  article_id,
+  setNewComment,
+  setCurrentAuthor,
+  setCurrentComments,
+}) {
+  const [isPosted, setIsPosted] = useState(false);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    postComment(newComment, currentAuthor, article_id).then((addedComment) => {
+      console.log(addedComment);
+      setCurrentComments((currentComments) => {
+        return [addedComment, ...currentComments];
+      });
+      setIsPosted(true);
```

Comment:
At the moment my comment is left in the comment box after it has been submitted. You could clear the state back to an empty string here.